CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Version 1.10.0-beta
 
-## 1.10.0-beta.7
```

Comment:
nit: I think we should keep the existing changelog entry for `beta.7` and then add a new one. Also the updating the changelog is currently part of the release process, so not something that needs to be updated in a specific feature PR.

Reply:
reverted